README.md

```diff
@@ -51,7 +51,9 @@ Usage: browser-run [OPTIONS]
 Options:
   --browser, -b  Browser to use. Always available: electron. Available if installed: chrome, firefox, ie, phantom, safari  [default: "electron"]
   --port         Starts listening on that port and waits for you to open a browser                                       
-  --static       Serve static assets from this directory                                                                 
+  --static       Serve static assets from this directory
+  --injectcss    Inject stylesheet links to the html header
+  --injectvdom   Inject virtual DOM entry points to the html body                                                                 
```

Comment:
`injectcss` sounds great to me, very handy, however I'm not sure about `injectvdom`.

Can you explain in which scenarios this is required? Keep in mind that we also have the `--input=html` option, which you can use to provide any kind of html you want. So we should only introduce command line flags for the most used options.

Reply:
If you pick up most of the known web framework such as [react.js](https://github.com/facebook/react), [riot](https://github.com/riot/riot), [vue](https://github.com/vuejs/vue) and few more, they follow the philosophy of having single entry for the virtual DOM. I want a quick solution rather than writing whole html template as option which can be bad if you keep using it around the cli.

Reply:
What about configuring the app so that:

```js
const root = process.env.NODE_ENV === 'test'
  ? 'document.body'
  : '#main'
app.mount(root)
```

And then run your tests with `NODE_ENV=test`?

Reply:
That's another solution by handling through javascript, it could be done that way. I think the injectvdom could be dropped if it to small of a detail. Anyway I will still utilize it, since by mounting my app to ```document.body``` pretty much break the css hierarchy and I'm incline to rewrite the whole css tree of just because the test does not support DOM entries.